### ISSUE TITLE
Palisade walls are to weak, even though there meant to be a effective defensive structure!

### ISSUE THREAD

**Issue by Krieger1999** [CleverRaven/Cataclysm-DDA#67069]
### Is your feature request related to a problem? Please describe.

Palisade walls are to weak compared to log house walls. Palisade walls have a min strenght value of 35 while the log wall has a min strenght value of 60, which is way to far apart, considering that palisades where commonly used as a quick, easy, cheap and strong, defence to build.
but currently there are longer to build, mostly though the deep pit and also take a higher skill requirement as well, than log walls.
So i wanted to change that so something that makes more sense for actuall palisades.

### Solution you would like.

1. Increase the min strenght value of the Palisade wall to 55 close to 60 from the log wall
and the palisade wall with embrasure to 45 and palisade gate to 45 as well. to simulate that palisade walls are not as strong as log house walls but still very sturdy and more time and cost efficent to use for defence.
2. Decrease the construction time from all 3 of them to 20 min. and change the requirement to to let them get build on a shallow pit and not deep pit, to further decrease there building time.
3. Decrease the skill requirement for all 3 of them to 2 farbrication and 2 survival, to simulate, that there were easy to build.

### Describe alternatives you have considered.

Alternatives would be to further adjust these values, to balance them even better. Iam open for suggestions.

**Comment by teapot156** [comment]
Agreed on all counts

**Comment by Procyonae** [comment]
So er... palisades are made with 3 logs each rn which doesn't sound that bad, except they're only 140cm long and remembering alot of that is going underground the palisade would be a short fence at best, don't know the best way around it the only longer things we have are beams https://cdda-guide.nornagon.net/item/wood_beam and long poles/planks which are made from beams but beams take 2 logs and either a long time or electric tools to craft which rather defeats the purpose

**Comment by Krieger1999** [comment]
> So er... palisades are made with 3 logs each rn which doesn't sound that bad, except they're only 140cm long and remembering alot of that is going underground the palisade would be a short fence at best, don't know the best way around it the only longer things we have are beams https://cdda-guide.nornagon.net/item/wood_beam and long poles/planks which are made from beams but beams take 2 logs and either a long time or electric tools to craft which rather defeats the purpose

Yes but log walls also only use 4 logs and have to fill the same space as the palisade wall anyway. If both use a shallow pit and like 4 logs than it should be fine.

**Comment by Procyonae** [comment]
How is that relevant, neither log walls nor palisades should use pits as I explained on the discord, pits are 1x1m square holes, shallow is a pitiful 25cm deep while the pit is 175cm deep, neither suitable for putting logs into. Palisades should have digging costs inbuilt to their construction and log walls could do the same or use a fence post as a base bc they just need something to brace against. 3 vs 4 logs isn't my argument, a log wall is logs stacked horizontally braced to logs in the ground so they only need to be 1m long to fit the tile, palisades are upright logs all wedged in the ground so they'd be less than the height of 1 log and 140cm is already too small for a wall. Just bc other things don't make sense doesn't mean we should add more.

**Comment by Krieger1999** [comment]
> 



> How is that relevant, neither log walls nor palisades should use pits as I explained on the discord, pits are 1x1m square holes, shallow is a pitiful 25cm deep while the pit is 175cm deep, neither suitable for putting logs into. Palisades should have digging costs inbuilt to their construction and log walls could do the same or use a fence post as a base bc they just need something to brace against. 3 vs 4 logs isn't my argument, a log wall is logs stacked horizontally braced to logs in the ground so they only need to be 1m long to fit the tile, palisades are upright logs all wedged in the ground so they'd be less than the height of 1 log and 140cm is already too small for a wall. Just bc other things don't make sense doesn't mean we should add more.

I just mean that if both use a shallow pit, than the log, which should have a height of 14dm and length of 1.89dm and width of 1.89dm, because of 1.89dm×1.89dm×14dm = 50dm^3 which is 50l

Then the log wall would have a total heigth of 4x1.89dm - 2.5dm = 5.06dm 
The palisade wall would have a height of 
14dm - 2.5dm = 11.5dm, which would be way taller then 5.06dm! So the palisade wall is actually the one who is more like a real wall right now. But would only be 5.67dm width right now in comparision to the log wall, who would be 14dm width!
Do you see how messed up these values are?! 😆
